aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -1114,6 +1117,13 @@ impl AptosVM {
                     self.0.mark_loader_cache_as_invalid();
                 };
 
+                if err.status_type() == StatusType::InvariantViolation {
+                    speculative_error!(
```

Comment:
it's great all those could be classified as errors :-)

Can we confirm by running one of Daniel's early abort PRs and making sure it finally doesn't log these Move-VM errors anymore? (I think the aptos-vm errors may still be flushed now, but commented there should be an easy fix)

Reply:
I think @danielxiangzl already tried it, but feel free to correct me

Reply:
Yes, I have tested the logging works as we want.